### ISSUE TITLE
Don't remove attributes when Downgrading to PHP 7

### ISSUE THREAD

**Issue by rafark** [rectorphp/rector#8908]
The DowngradeAttributeToAnnotationRector seems to remove attributes that are not from Symfony or Nette. Attributes should not be removed because they are fully backward compatible.

In my case, I need to keep the native #[AllowDynamicProperties] and #[ReturnTypeWillChange] because the code I'm downgrading will be executed in both PHP 7 AND 8+ (and dynamic properties are deprecated and therefore I'm getting deprecation warnings). And since attributes are just comments, they are compatible with php 7 so there's really no point in removing them.

in `downgrade-php80.php` I'm manually commenting this line (since I don't use Symfony):

`
/*$rectorConfig->ruleWithConfiguration(DowngradeAttributeToAnnotationRector::class, [
        // Symfony
        new DowngradeAttributeToAnnotation('Symfony\\Contracts\\Service\\Attribute\\Required', 'required'),
        // Nette
        new DowngradeAttributeToAnnotation('Nette\\DI\\Attributes\\Inject', 'inject'),
        // Jetbrains\PhpStorm\Language under nette/utils
        new DowngradeAttributeToAnnotation('Jetbrains\\PhpStorm\\Language', 'language'),
    ]);*/
`

In conclusion, as I mentioned, my solution would be to not remove attributes (symfony attributes can still be converted to annotations for those who need that feature).

**Comment by TomasVotruba** [comment]
Thank you for your report!

We'll need an isolated failing demo link from: http://getrector.com/demo,
that way we can reproduce the bug.

**Comment by rafark** [comment]
Here it is: https://getrector.com/demo/e8ff08f9-04d4-456f-9223-440b2090f2ea

There's no reason to remove #[AllowDynamicProperties] since it is just a comment in php 7. So my proposal is to not remove unknown (to Rector) attributes.
